.golangci.yaml

```diff
@@ -71,3 +71,7 @@ linters-settings:
   # Never have naked return ever
   nakedret:
     max-func-lines: 1
+  staticcheck:
+    checks:
+      - all
+      - '-SA3000' # disable false positive related to m.Run() and os.Exit. That's not needed since Go 1.15.
```

Comment:
this shouldn’t be needed with latest golangci-lint, no?

Reply:
Yeah, we upgraded to 1.60.3 yesterday, this part should no longer be needed. I just rebased over the latest main and dropped that change.